.github/ISSUE_TEMPLATE/onboarding_argocd.md

```diff
@@ -45,4 +45,4 @@ Select all that apply:
 
 ---
 
-If you don't want to wait for the maintainers to notice this issue, you can do the changes yourself. Please follow the [docs/onboarding_team_to_argo_cd.md](https://github.com/operate-first/argocd-apps/blob/main/docs/onboard_team_to_argocd.md) and/or the [docs/add_application.md](https://github.com/operate-first/argocd-apps/blob/main/docs/add_application.md) guides.
+If you don't want to wait for the maintainers to notice this issue, you can do the changes yourself. Please follow the [docs/onboarding_team_to_argo_cd.md](../../docs/onboarding_to_argocd.md) and/or the [docs/add_application.md](https://github.com/operate-first/argocd-apps/blob/main/docs/add_application.md) guides.
```

Comment:
The relative URL seems not to be working in issue templates. :/ You'd need to use an absolute url here. I found the hard way. See https://github.com/operate-first/support/issues/45

Reply:
updated